db/active_replicator_checkpointer_test.go

```diff
@@ -43,6 +46,12 @@ func genProcessedForTest(t testing.TB, seqs ...string) map[SequenceID]struct{} {
 
 func TestCheckpointerSafeSeq(t *testing.T) {
 
+	stats, err := base.SyncGatewayStats.NewDBStats(t.Name(), false, false, false, nil, nil)
+	replicationStats, err := stats.DBReplicatorStats(t.Name())
+	require.NoError(t, err)
+	replicatorConfig := &ActiveReplicatorConfig{}
+	replicatorConfig.ReplicationStatsMap = replicationStats
+
```

Comment:
Initialising stats this way means each subtest shares the same stats, which might make future assertions tricky and adds a lot of test boilerplate.

Would it be better to move this code into the test function and set `c.stats` there?

Reply:
https://github.com/couchbase/sync_gateway/pull/6098/commits/8046b642151d0b4a0e7dfb2ddeb343935f782201